src/suites/cts/gpu_test.ts

```diff
@@ -114,6 +114,44 @@ export class GPUTest extends Fixture {
     });
   }
 
+  checkCopyImageBitmapResult(
+    src: GPUBuffer,
+    expected: ArrayBufferView,
+    width: number,
+    height: number,
+    rowPitch: number,
+    bytesPerPixel: number
+  ): void {
```

Comment:
This function duplicates some of the code from expectContents. Could they be refactored to share some code?

Sorry I don't have a specific recommendation on how to do this, I don't have time to dive into it right now.

Reply:
Sure, and refactor makes sense. I'll address this.

Reply:
Did some simple refactor to reuse some common code.  A further deep refactor still under consideration.